src/ZenithAngleCalc.jl

```diff
@@ -1,72 +1,68 @@
 export instantaneous_zenith_angle, daily_zenith_angle
 
-# mean anomaly at vernal equinox, radians (eq 3.11)
-function mean_anomaly_vernal_equinox(ϖ::FT, e::FT) where {FT <: Real}
-    β = (FT(1)-e^FT(2))^FT(0.5)
-    M_v = -ϖ + (e+FT(0.25)*e^FT(3))*(FT(1)+β)*sin(ϖ)
-            - FT(0.5)*e^FT(2)*(FT(0.5)+β)*sin(FT(2)*ϖ)
-            + FT(0.25)*e^FT(3)*(FT(1)/FT(3)+β)*sin(FT(3)*ϖ)
-    M_v = mod(M_v, FT(2)*FT(π))
-    return M_v
-end
-
 # true anomaly, radians (eq 3.8)
 function true_anomaly(MA::FT, e::FT) where {FT <: Real}
-    TA = MA + (FT(2)*e - FT(0.25)*e^FT(3))*sin(MA) 
-            + FT(1.25)*e^FT(2)*sin(FT(2)*MA) 
-            + FT(13)/FT(12)*e^FT(3)*sin(FT(3)*MA)
-    TA = mod(TA, FT(2)*FT(π))
-    return TA
+    TA = MA + (2*e - FT(1/4)*e^3)*sin(MA) 
+            + FT(5/4)*e^2*sin(2*MA) 
+            + FT(13/12)*e^3*sin(3*MA)
+    return mod(TA, FT(2π))
 end
 
 # equation of time, radians
 function equation_of_time(e::FT, MA::FT, γ::FT, ϖ::FT) where {FT <: Real}
-    Δt = -FT(2)*e*sin(MA) + tan(γ/FT(2))^FT(2)*sin(FT(2)*(MA + ϖ)) 
-    Δt = mod(Δt+π, FT(2)*FT(π))-π
+    _Δt = -2*e*sin(MA) + tan(γ/2)^2*sin(2*(MA + ϖ));
+    return mod(_Δt+FT(π), FT(2π)) - FT(π)
 end
 
 # calculate the distance, declination, and hour angle (at lon=0)
-function distance_declination_hourangle(::Type{FT}, date::DateTime, param_set::APS, eot_correction::Bool) where {FT <: Real}
+function distance_declination_hourangle(::Type{FT},
+                                        date::DateTime,
+                                        param_set::APS,
+                                        eot_correction::Bool,
+                                        milankovitch::Bool) where {FT <: Real}
     Ya::FT = year_anom(param_set)
     day_length::FT = Planet.day(param_set)
     AU::FT = astro_unit()
-
     _epoch::FT = epoch(param_set)
     M0::FT = mean_anom_epoch(param_set)
-    ϖ0::FT = lon_perihelion_epoch(param_set)
 
-    γ::FT = obliq_epoch(param_set)
-    ϖ::FT = lon_perihelion(param_set)
-    e::FT = eccentricity_epoch(param_set)
+    t0_years = _epoch / Ya; # s --> years
```

Comment:
If _epoch were a date string, it could just be converted to Julian day here, in the same way that t_years is calculated.